renovate.json5

```diff
@@ -3,7 +3,7 @@
   ignorePresets: [':prHourlyLimit2'],
   semanticCommits: true,
   dependencyDashboard: true,
-  automerge: false,
```

Comment:
@erezrokah should we also add this to - https://github.com/netlify/node-template/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/renovate.json5 - or are we going to do it on a separate PR?

Reply:
Good point - I was thinking of only enabling it for this repo and note for the template itself.
I'll add it for the template too.